bin/init.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var filesToCopy = [
+  '.travis.yml',
+  'gulpfile.js'
+];
+
+function copyFileSync(srcFile, destFile, encoding) {
```

Comment:
Is opening the fie and reading its contents, then writing it back out really the best way to copy a file?